static/src/stylesheets/module/content/_live-blog.scss

```diff
@@ -271,6 +271,27 @@ $block-padding-right: $gs-gutter;
 
     .block-elements {
 
+        /* Bylines
+           ========================================================= */
+        .liveblog-block-byline {
+            border-bottom: 1px dotted #dcdcdc;
+            padding-bottom: 8px;
+        }
+        .liveblog-block-byline__img {
+            display: inline;
+            border-radius: 50%;
+            max-width: 36px;
+            vertical-align: middle;
+            margin-right: 4px;
+        }
+        .liveblog-block-byline__name {
+            font-family: 'EgyptianText', Georgia;
+            font-weight: normal;
+            font-size: 16px;
```

Comment:
Similar to lengths, type is defined by a bunch of presets in https://github.com/guardian/guss-typography. there's a graphic demoing the sizes/look etc at https://raw.githubusercontent.com/guardian/guss-typography/master/font-scale.png, but for this, you'd use `@include fs-bodyCopy(2)`.